communication/slack-config/restrictions.yaml

```diff
@@ -10,6 +10,11 @@ restrictions:
       - ".*"
   - path: template.yaml
     template: true
+  - path: "sig-architecture/*.yaml"
+    channels:
+      - ".*"
```

Comment:
it's a little frustrating since there isn't a naming convention, but we should probably set these up with a regex to match the channels being moved over
```
    channels:
      - "^enhancements$"
      - "^sig-architecture$"
      - "^k8s-code-organization$"
      - "^k8s-conformance$"
```